demo/index.html

```diff
@@ -13,6 +13,9 @@
       import '@polymer/iron-demo-helpers/demo-snippet';
     </script>
     <script type="module" src="../src/ods-inputoptions.js"></script>
```

Comment:
ods-inputoptions should be removed from this version. This is a BREAKING CHANGE and there is no need to support this version any longer.